src/respo_alerts/comp/alerts.cljs

```diff
@@ -19,7 +20,14 @@
  (assert (fn? on-read!) "require a callback function")
  (let [state (or (:data states) {:show? false})]
    (span
-    {:style {:cursor :pointer}, :on-click (fn [e d! m!] (m! (assoc state :show? true)))}
+    {:style {:cursor :pointer},
+     :on-click (fn [e d! m!]
+       (m! (assoc state :show? true))
+       (js/setTimeout
+        (fn []
+          (let [target (.querySelector js/document (str "." schema/confirm-button-name))]
+            (if (some? target) (.focus target))))
+        50))}
```

Comment:
I have several questions:

* Is there a better way than `setTimeout`?
* Why not `getElementsByClassName`?
* Is it possible that there are multiple elements with the class?
* What if the caller wants the cancel button to be default? (e.g. "Destroy the world?")

Reply:
> Is there a better way than setTimeout?

No, I can think of one. I have to wait next loop of rendering. Currently there's no API got that in Respo.

> Why not `getElementsByClassName`?

I believe `d.querySelector()` is a more general solution. In this case I don't even worry about performance.

> Is it possible that there are multiple elements with the class?

Unlikely in this app. I'm not going to pop 2 of them.

> What if the caller wants the cancel button to be default? (e.g. "Destroy the world?")

Mouse click. Haven't think about navigating with keyboard yet. Only mouse click by now.

Reply:
> Unlikely in this app. I'm not going to pop 2 of them.

Isn't this a library?

Reply:
Yes, it's a library. I was saying I don't pop 2 of them, in the business logics.

Reply:
`alert` `prompt` `confirm` are browser APIs, which are blocking the thread. This library is trying to replacing them with Respo. So they are not intended to have multiple instances in the mean time.